package.json

```diff
@@ -11,8 +11,8 @@
     "cy:open": "yarn cypress open"
   },
   "engines": {
-    "node": "^12.20.0",
-    "npm": "^6.14.8"
+    "node": "16.x",
+    "npm": "6.x"
```

Comment:
Out of curiosity, any reason to no bring NPM to most recent version 8?

Reply:
Because we are using `yarn` instead of `npm`. We can remove this and specify the desired yarn version. What do you think?

Reply:
![jake-what](https://user-images.githubusercontent.com/301545/142900765-ab9e1036-c9ad-4bf9-8262-c8ed5ebf6cfe.gif)

That's awesome. We're using `yarn` but we dont have it as project dependency in `package.json`? 😱 

I'm full on to remove `npm` and add `yarn` !
cc @miltonbsn any possible blocker here?

Reply:
I don't see a blocker! let's all in npm -> yarn 😃 

Amazing job again @oicramps ⭐⭐⭐⭐⭐

Reply:
I've replaced `npm` with `yarn`. Can you guys test it one more time, please? Thank you!